src/app/utils/xml-utils.ts

```diff
@@ -36,3 +36,141 @@ export function replaceNotWordChar(textContent: string) {
 export function removeSpaces(textContent: string) {
   return textContent.replace(/\s/g, '');
 }
+
+/**
+ * It removes excessive spaces, any tabulation, new lines and non-word characters
+ * @param textContent string
+ * @returns string
+ */
+export function normalizeSpaces(textContent: string) {
+  return textContent.replace(/[\s]{2,}|\n|\t|\r/g, ' ').trimStart().trimEnd();
+}
+
+/**
+* Significant text sometimes is split inside two or more text evt-element inside the main one, especially when it contains new line characters.
+* This function returns a string with all the text element chained
+* @param n XMLElement
+* @returns string
+*/
+export function chainFirstChildTexts(elem: XMLElement): string {
+  if (elem === undefined) { return ''};
+  const evtTextElements = {
+    '#text': 'nodeValue',
+    'p': 'textContent',
+  };
+  const evtTextComplexElements = ['choice', 'app', 'l', 'quote', 'p', 'lg'];
```

Comment:
EVT logic shouldn't stay in utility generic function. I would probably try to put it in the parser, since it is strictly connected to the xml tei parser and use a property passed to the function to indicate complex elements. @giuliac89 @szenzaro what do you think? How would you do?